plugins/cost-insights/src/components/CostInsightsPage/CostInsightsPage.tsx

```diff
@@ -211,7 +209,7 @@ export const CostInsightsPage = () => {
         </Typography>
       </Box>
       <Box minHeight={40} maxHeight={60} display="flex">
-        {!!flags.get('cost-insights-currencies') && (
```

Comment:
Existing usage like this is not forwards compatible with all possible state additions, which is why there's only an `isActive()` now